src/napari/_vispy/visuals/interaction_box.py

```diff
@@ -75,7 +75,7 @@ def set_data(
             The bottom right corner of the interaction box.
         handles : bool
             Whether to show the handles of the interaction box.
-        selected : int | None
+        selected : int | slice | None
```

Comment:
```suggestion
        selected : int | slice[int, int, int] | None
```

Reply:
In `builtins.py` it is referred as a generic from what I can see but I might be mistaken:

```py
@final
class slice(Generic[_StartT_co, _StopT_co, _StepT_co]):
```

Reply:
Backtracking from SO to this [PR](https://github.com/python/typeshed/pull/11637/files) to confirm that it's been annotated as Generic a while back